Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealityConfigurationProfileInspector.cs

```diff
@@ -252,6 +260,17 @@ public override void OnInspectorGUI()
                 EditorGUILayout.PropertyField(teleportDuration);
             }
 
+            // Spatial Awareness System configuration
+            GUILayout.Space(12f);
+            EditorGUILayout.LabelField("Spatial Awareness System Settings", EditorStyles.boldLabel);
+            EditorGUILayout.PropertyField(enableSpatialAwarenessSystem);
+
+            if (enableSpatialAwarenessSystem.boolValue)
```

Comment:
Please remove this so we don't hide it when disabled.

Reply:
every system in the core profile behave this way